docs/getting-started.rst

```diff
@@ -433,6 +431,7 @@ And execute the following script:
     )
     tracer_provider = TracerProvider()
     trace.set_tracer_provider(tracer_provider)
+    span_processor = BatchExportSpanProcessor(span_exporter)
```

Comment:
An aside, but should we be using SimpleSpanProcessors in our examples? I feel like being available as an example is the point.

Reply:
That's a good question. I personally prefer to use the batch one because I don't want to create the impression that the simple is the default choice. Maybe we could adopt the convention of using `SimpleSpanProcessor` for the console span exporter and the `BatchSpanProcessor` for the others.